store/localstore/snapshot.go

```diff
@@ -32,9 +31,27 @@ var (
 
 type dbSnapshot struct {
 	db      engine.DB
+	rawIt   engine.Iterator
 	version kv.Version // transaction begin version
 }
 
+func (s *dbSnapshot) internalSeek(startKey []byte) (engine.Iterator, error) {
+	if s.rawIt == nil {
+		var err error
+		s.rawIt, err = s.db.Seek([]byte{0})
```

Comment:
No, we can't.
if you use db.Seek(startKey) directly the rawIt only contains keys in the given range: (startKey, +inf).